firmware/cartridges/cartridge.c

```diff
@@ -31,6 +31,7 @@
 #include "super_snapshot_5.c"
 #include "easyflash.c"
 #include "comal80.c"
+#include "warpspeed.c"
```

Comment:
I tend to order the includes in the same order as the CRT number. That makes it a bit easier (for me at least)